astropy/table/column.py

```diff
@@ -519,9 +519,13 @@ def __new__(
         unit=None,
         format=None,
         meta=None,
-        copy=False,
+        copy=COPY_IFF_NEEDED,
```

Comment:
For the "band aid" solution, I think this would need to remain `False`, with the below band-aid patch remaining in place.